### PR TITLE
Patch 2 - Veloviewer Map Flipper

### PR DESCRIPTION
**Pull request by KamensekD**
Please check if it works in this simple way. Might not, since it works if you copy the "javascript:..." string into a bookmark and click on it, but doesn't seem work if you copy the string into browsers url field ?!?
check: http://veloviewer.com/mapFlipper   and   https://s3.amazonaws.com/s3.veloviewer.com/js/vv.mapFlipper.js

### CODE REVIEW

File: hook/extension/js/modifiers/RemoteLinksModifier.js [hunk 1]
```diff
@@ -49,7 +57,11 @@ RemoteLinksModifier.prototype = {
         htmlRemoteViewForActivity += "<ul>";
         jQuery.each(remoteViewActivityLinksArray, function() {
             htmlRemoteViewForActivity += "<li>";
-            htmlRemoteViewForActivity += "<a data-menu='' target='_blank' style='color: #333;' href='" + this[1] + pageView.activity().id + this[2] + "'>" + this[0] + "</a>";
+            if (this[2] == 'mapflipper') {
+                htmlRemoteViewForActivity += "<a data-menu='' target='_blank' style='color: #333;' href='" + this[1] "'>" + this[0] + "</a>";
```
Comment (thomaschampagne) [inline hook/extension/js/modifiers/RemoteLinksModifier.js]: Must be fixed with:
```
htmlRemoteViewForActivity += "<a data-menu='' target='_blank' style='color: #333;' href='" + this[1] + "'>" + this[0] + "</a>";
```

**Comment by thomaschampagne** [comment]
Thanks for help.

I'm not able to get your pull request working. Even with the following fix:

```
htmlRemoteViewForActivity += "<a data-menu='' target='_blank' style='color: #333;' href='" + this[1] + "'>" + this[0] + "</a>";
```

**Comment by KamensekD** [comment]
I have another idea how to make it work. I have to work on it and I'll try to make another pull request soon.

**Comment by thomaschampagne** [comment]
Thanks ;)

**Comment by KamensekD** [comment]
Hey Thomas! Please take a look at
https://github.com/KamensekD/StravaPlus/commit/b2ee5ce963bac4f6e4ed6c9a5028158bdb1ff8aa
I'm confident this might work now

**Comment by KamensekD** [comment]
I managed to build and test my own fork and can report that it does work :) You have to select "Satellite view" in map itself:
flipper
You should take care of visually formatting the Map Flipper links, though.

Regards!
Dejan
sLOVEnia

**Comment by KamensekD** [comment]
This pull request is replaced with https://github.com/thomaschampagne/StravaPlus/pull/72

**Pull request closed without merging**